python/src/aiconfig/editor/client/src/components/prompt/ModelSelector.tsx

```diff
@@ -40,12 +40,15 @@ export default memo(function ModelSelector({
     <Autocomplete
       placeholder="Select model"
       limit={100}
+      className="ghost"
+      variant="unstyled"
```

Comment:
Just noting that we've had user feedback about unstyled inputs before (discord comment about parameter name input which was previously unstyled in lastmile workbooks)

Reply:
These ghost buttons behave differently. hover shows an input outline and cursor changes.